### ISSUE TITLE
Package installation broken since 1.2.0 (Cannot read property 'logoutUser' of null)

### ISSUE THREAD

**Issue by drahoslove** [customerly/Customerly-ReactNative-SDK#12]
This was added into `package.json` in the 1.2.0 version.
```
  "files": [
    "src"
  ],
 ````
 
 It is causing all the files outside of the `src` directory, to be excluded from the installed package.
 https://docs.npmjs.com/cli/v10/configuring-npm/package-json#files
 
Because of this, the `RNCustomerly` pod is not being installed, and `RNCustomerly` imported from `NativeModules` inside of [src/RNCustomerly.js](https://github.com/Customerly/Customerly-ReactNative-SDK/blob/master/src/RNCustomerly.js) is `null`.

Nothing goes visibly wrong during the installation of packages, or installation of pods. But when the app starts, following error is thrown:
```
ERROR TypeError: Cannot read property 'logoutUser' of null, js engine: hermes
```

Temporary solution for me was to define the dependency in my `package.json` as

```
"react-native-customerly": "https://github.com/Customerly/Customerly-ReactNative-SDK.git#834af7385eda6798cef7d7b589a84016ad333626",
```
instead of
```
"react-native-customerly": "1.2.1",
```

**Comment by GiorgioBertolotti** [comment]
Hey @drahoslove, thanks for reporting the issue and providing a possible solution!

Unfortunately the next week I'll be very busy with an event, so I will try to check the problem in my spare time and then I'll reach you out!
Thanks again 😄

**Comment by GiorgioBertolotti** [comment]
@drahoslove I removed the `files` declaration in package.json, hope it fixes the problem!
I'll wait to release 1.2.2 because I want to fix #11 within the next days.

Thanks again for reporting the issue 😄

**Comment by drahoslove** [comment]
Great, thank you for the fix @GiorgioBertolotti 👏🏼
